index.html

```diff
@@ -24,6 +24,15 @@
 
   </script>
   <link type="text/plain" rel="author" href="humans.txt" />
+  
+  <meta property="og:locale" content="en_US" />
```

Comment:
Perhaps group all these meta tags on the top with the other meta tags and add a line comment for Facebook Open Graph?